pkg/task/runner.go

```diff
@@ -111,21 +112,26 @@ func (m *TaskGroupRunner) isTaskIDUnique(identity string) (unique bool) {
 // NOTE:
 //  This is just the planning for rollback & not actual rollback.
 // In the events of issues this planning will be useful.
-func (m *TaskGroupRunner) planForRollback(te *taskExecutor, objectName string) (err error) {
-	// the entire rollback plan is encapsulated in the task itself
-	rte, err := te.asRollbackInstance(objectName)
-	if err != nil {
-		return
-	}
+func (m *TaskGroupRunner) planForRollback(te *taskExecutor, objectName string) error {
+	objNames := strings.Split(objectName, ",")
 
-	if rte == nil {
-		// this task does not need a rollback or
-		// can not be rollback-ed in-case of above error
-		return
+	// plan the rollback for all the objects that got created
+	for _, oName := range objNames {
```

Comment:
Can we have `name` instead of `oName` for the sake of consistency.

Reply:
yup. I will change.